src/Widgets/PopoverWidget.vala

```diff
@@ -47,10 +45,8 @@ public class Power.Widgets.PopoverWidget : Gtk.Box {
                 device_separator = new Wingpanel.Widgets.Separator ();
                 pack_start (device_separator);
             }
-
-            debug ("show brightness slider");
-            screen_brightness = new ScreenBrightness ();
-            pack_start (screen_brightness);
+        screen_brightness = _screen_brightness;
```

Comment:
Instead, you can name both `screen_brightness` and do `this.screen_brightness = screen_brightness;`